glass/.styleguide

```diff
@@ -24,6 +24,7 @@ includeOtherLibs {
   ^fmt/
   ^frc/
   ^imgui
+  ^networktables/
```

Comment:
What's this?

Reply:
It treates header includes starting with `networktables/` as "other libraries" during sorting.
https://github.com/wpilibsuite/styleguide/blob/main/wpiformat/README.rst#styleguide

Reply:
Why is there also `networktables` in addition to `ntcore`?

Reply:
The header folder name has always been networktabes/, not ntcore/. The `^ntcore/` entry should be removed.

Reply:
It's `^ntcore`, not `^ntcore/`, which means it shouldn't be removed--this is the item that covers `ntcore_cpp.h` and kin.